diesel/src/migrations/mod.rs

```diff
@@ -132,38 +132,37 @@ pub fn mark_migrations_in_directory<Conn>(conn: &Conn, migrations_dir: &Path)
 ///
 /// See the [module level documentation](index.html) for information on how migrations should be
 /// structured, and where Diesel will look for them by default.
-pub fn revert_latest_migration<Conn>(conn: &Conn) -> Result<String, RunMigrationsError> where
+pub fn revert_latest_migration<Conn>(conn: &Conn, migrations_dir: &Path) -> Result<String, RunMigrationsError> where
```

Comment:
This is the correct answer. And while breaking changes are allowed since we're still pre-1.0, I would like to keep this function as is. Most of these functions have the `do_thing` and `do_thing_in_directory`